homeUi/src/main/kotlin/com/gravatar/app/homeUi/presentation/home/profile/ProfileScreen.kt

```diff
@@ -32,7 +39,23 @@ internal fun ProfileScreen(uiState: ProfileUiState) {
         } else {
             uiState.profile.let { profile ->
                 if (profile != null) {
-                    ProfileHeader(profile = profile, modifier = Modifier.padding(16.dp))
+                    Column {
+                        Spacer(modifier = Modifier.height(16.dp))
+
+                        ProfileHeader(profile = profile, modifier = Modifier.padding(horizontal = 16.dp))
+
+                        Spacer(modifier = Modifier.height(16.dp))
```

Comment:
```suggestion
                        ProfileHeader(profile = profile, modifier = Modifier.padding(16.dp))
```

⛏️ I believe this would result in the same UI, any reason to go with spacers here?